### ISSUE TITLE
Campaign banners are not contained within a landmark

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14882]
**Describe the bug**

When a campaign banner is added to the top of the home page, it's not automatically contained within any landmark. While admins are in control of the HTML of the banner and _could_ use a landmark, we should support admins in creating accessible content rather than expect everyone to have the HTML/accessibility knowledge to do this for each campaign.

Since campaign banners support the main content of the home page, but aren't part of the home feed itself, I would suggest we place all hero campaign banners inside an `<aside>` element with the aria-label "Campaign banner" (a name specific to the individual campaign would be better, but at least this would convey the nature of the content).

**To Reproduce**

- Create a campaign banner locally ([see the admin docs for guidance](https://admin.forem.com/docs/advanced-customization/config/campaigns))
- Visit the home page
- Run axe dev tools or similar, and note the error that "All content must be contained within landmarks"

**Expected behavior**

- The error should not be visible in an axe report
- A screen reader user should be able to skip directly to the campaign banner when navigating by landmarks

**Screenshots**

![screenshot of the DEV homepage with a Hacktoberfest campaign banner across the top of the page. The axe dev tools is open and an error is highlighted, noting that the banner is not contained within a landmark](https://user-images.githubusercontent.com/20773163/135642107-e5a97ef3-9e80-465a-802d-844724e84b6f.png)

**Comment by pramodrhegde** [comment]
@aitchiss I would love to make a contribution here. please assign this to me. thanks!

**Comment by crisbnp** [comment]
Can I work on this one please?

**Comment by benhalpern** [comment]
@crisbnp I'll assign this to you since I know @pramodrhegde got at least one other one.

**Comment by aitchiss** [comment]
Closed in #15034
